packages/build/src/homebrew/generate-formula.spec.ts

```diff
@@ -29,8 +29,8 @@ class Mongosh < Formula
   end
 
   test do
-    assert_match "ECONNREFUSED 0.0.0.0:1", shell_output("#{bin}/mongosh \"mongodb://0.0.0.0:1\" 2>&1", 1)
-    assert_match "#ok#", shell_output("#{bin}/mongosh --nodb --eval \"print('#ok#')\"")
+    assert_match "ECONNREFUSED 0.0.0.0:1", shell_output("#{bin}/mongosh \\"mongodb://0.0.0.0:1\\" 2>&1", 1)
```

Comment:
I'm not sure how this worked before. Either the \ was useless because \" would just be " or it should have been \\? We can also just ignore the linting for this file.

Reply:
It’s a test fixture, right? We’re not actually running this code.

I’d have a mild preference for prefixing this string with `String.raw` instead of worrying too much about escaping into it, but that’s definitely not required, feel free to leave this as-is.